libbindgen/tests/headers/disable-namespacing.hpp

```diff
@@ -0,0 +1,9 @@
+// bindgen-flags: --disable-name-namespacing
+
+namespace foo {
+namespace bar {
+
+typedef int Baz;
```

Comment:
We don't, and rustc generates a duplicated name error. I think this is fair to do in this case, given we warn properly.

To be fair, I pretend this option to not be widely used in general. I want it to be mostly a vehicle to first being able to build stylo with master bindgen again, and then updating stylo to use namespaces properly.